Makefile

```diff
@@ -2521,7 +2521,7 @@ swift_deploy_cleanup: ## cleans up the service instance, Does not affect the ope
 .PHONY: certmanager
 certmanager: export NAMESPACE=$(if $(findstring 4.10,$(OCP_RELEASE)),openshift-cert-manager,cert-manager)
 certmanager: export OPERATOR_NAMESPACE=$(if $(findstring 4.10,$(OCP_RELEASE)),openshift-cert-manager-operator,cert-manager-operator)
-certmanager: export CHANNEL=$(if $(findstring 4.10,$(OCP_RELEASE)),tech-preview,stable-v1.11)
+certmanager: export CHANNEL=$(if $(findstring 4.10,$(OCP_RELEASE)),tech-preview,$(if $(findstring 4.15,$(OCP_RELEASE)),stable-v1.13,stable-v1.11))
```

Comment:
would be better to cleanup 4.10 checks itself here and above and in cert_manager_cleanup to be more cleaner, which is already EOL, also with other part of code it will not work on older OCPs

Reply:
yes sure